docs/UserGuide.md

```diff
@@ -184,23 +184,23 @@ An "alphanumeric" character is a character that can be either a letter (A to Z,
 
 ## Patient Management Features
 
-This is the list of features that help you manage the records of patient:
+This is the list of features that help you manage the records of patients:
```

Comment:
record should be singular in this case?

Reply:
reworded to say `This is the list of features that help you manage patient records:`